packages/pigeon/lib/swift_generator.dart

```diff
@@ -969,6 +945,9 @@ String _swiftTypeForDartType(TypeDeclaration type) {
 
 String _nullsafeSwiftTypeForDartType(TypeDeclaration type) {
   final String nullSafe = type.isNullable ? '?' : '';
+  if (type.baseName == 'Map') {
```

Comment:
Maybe a comment in the code here for why Map needs special handling? It's not obvious to me what this change is for.